src/training/character_specific/mod.rs

```diff
@@ -0,0 +1,12 @@
+mod bowser;
+
+/**
+ * Checks if the current status matches the expected status
+ */
+pub fn check_status(current_status: i32, expected_status: i32) -> bool {
+    if bowser::check_up_b(current_status, expected_status) {
```

Comment:
We may also need to check against `app::utility::get_kind() == FIGHTER_STATUS_KIND_BOWSER`. Necessary because the consts specific to Bowser have *values* that may be shared by other characters for status kinds.